### ISSUE TITLE
Clicking on a folder icon doesn't navigate to folder

### ISSUE THREAD

**Issue by kibhub** [tutao/tutanota#5428]
If I click on a folder's icon which has no subfolders - for both custom folders and default ones - nothing happens. I have to click on the text or empty area of the folder item/line.

**Comment by charlag** [comment]
I am not sure this is a bug, at least the icon has a consistent function. Need to discuss.

**Comment by kibhub** [comment]
I agree that this is not necessarily a bug but it is not obvious if you do not have any subfolders.
